src/osip-core/src/test/java/edu/kit/pse/osip/core/io/files/ClientSettingsWrapperTest.java

```diff
@@ -53,6 +63,55 @@ public void setUp() throws Exception {
         wrapper = new ClientSettingsWrapper(tempTestFile);
     }
     
+    private void initNullWrapper() throws IOException {
+        tempTestFileNullEntry = File.createTempFile("testClientSettingsTempNullEntry", ".properties");        
+        PrintWriter outStreamNullEntry = new PrintWriter(tempTestFileNullEntry);        
+        outStreamNullEntry.print("fetchIntervall=80");
+        outStreamNullEntry.println();
+        outStreamNullEntry.print("overflowAlarmm_" + (TankSelector.valuesWithoutMix()[0]) + "=" + "false");
+        outStreamNullEntry.println();
+        outStreamNullEntry.print("underflowAlarmm_" + (TankSelector.valuesWithoutMix()[0]) + "=" + "false");
+        outStreamNullEntry.println();
+        outStreamNullEntry.print("overheatingAlarmm_" + (TankSelector.valuesWithoutMix()[0]) + "=" + "false");
+        outStreamNullEntry.println();
+        outStreamNullEntry.print("undercoolingAlarmm_" + (TankSelector.valuesWithoutMix()[0]) + "=" + "false");
+        outStreamNullEntry.println();
+        outStreamNullEntry.print("tempDiagramm_" + (TankSelector.valuesWithoutMix()[0]) + "=" + "false");
+        outStreamNullEntry.println();
+        outStreamNullEntry.print("fillLevelDiagramm_" + (TankSelector.valuesWithoutMix()[0]) + "=" + "false");
+        outStreamNullEntry.println();
+        outStreamNullEntry.print("serverHostnamee_" + (TankSelector.valuesWithoutMix()[0]) + "=" + "hostNAME");
+        outStreamNullEntry.println();
+        outStreamNullEntry.print("serverPortt_" + (TankSelector.valuesWithoutMix()[0]) + "=" + "4242");
+        outStreamNullEntry.println();
+        outStreamNullEntry.close();
+        wrapperNullEntry = new ClientSettingsWrapper(tempTestFileNullEntry);
+    }
+       
+    private void initInvalidWrapper() throws IOException {
+        tempTestFileInvalidEntry = File.createTempFile("testClientSettingsTempInvalidEntry", ".properties");        
+        PrintWriter outStreamInvalidEntry = new PrintWriter(tempTestFileInvalidEntry);        
+        outStreamInvalidEntry.print("fetchInterval=8G0AER");
```

Comment:
Hier ebenfalls direkt println verwenden und TankSelector.valuesWithoutMix()[0] in einer Variable speichern.